### PR TITLE
Pr/494

### PR DESCRIPTION
**Pull request by red031000**
bringing this back because force push closed it

### CODE REVIEW

File: HTML/Ultimate Cipher condensed interactive (Fang).html [hunk 1]
```diff
@@ -0,0 +1,1861 @@
...
+							<code>...---xx<br>.-x.-x.-</code>
+						</p>
+					</li>
+					<li>Step 2: # of batteries is odd
+						<p style="text-align:center">
+							<code>
+							11111111122222222233333333<br>
+							11122233311122233311122233<br>
+							12312312312312312312312312
+						</p>
```
Comment (red031000) [inline HTML/Ultimate Cipher condensed interactive (Fang).html]: please close the code tag before closing the p tag

File: HTML/Ultimate Cipher condensed interactive (Fang).html [hunk 2]
```diff
@@ -0,0 +1,1861 @@
...
+			<div class="page-header">
+				<span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
+				<span class="page-header-section-title">Ultimate Cipher</span>
+			</div>
+			<div class="page-content">
+				<h2>On the Subject of the Green Cipher</h2>
+				<p class="flavour-text">Original Green Cipher manual can be found 
+					<a href="https://ktane.timwi.de/HTML/Green%20Cipher.html">here</a>.</p>
+				<p>This cipher contains Homophonic, Columnar Transposition and Mechanical Cipher.</p>
+				<font size = "2px">
```
Comment (red031000) [inline HTML/Ultimate Cipher condensed interactive (Fang).html]: font tags are not valid html5, please use css in style tags in the head instead

File: HTML/Ultimate Cipher condensed interactive (Fang).html [hunk 3]
```diff
@@ -0,0 +1,1861 @@
...
+				<font size="2">
+				<ul>
+					<li>Step 1: Morbit Cipher (Keyword isn't effected)</li>
+					<li>Step 3: Hill Cipher</li>
+					<li>Step 2: Trifid Cipher</li>
+				</ul>
+				</font>
+			<table class = "alpha" style="width:55%;float:left">
+			<tbody>
+			<tr><th>In</th><th>AND</th><th>OR</th><th>XOR</th><th>NAND</th><th>NOR</th><th>XNOR</th><th>-></th><th><-</th></tr>
```
Comment (red031000) [inline HTML/Ultimate Cipher condensed interactive (Fang).html]: please use &lt; and &gt; for less than and greater than symbols

Comment (Limeboy0603) [reply HTML/Ultimate Cipher condensed interactive (Fang).html]: that looks more like a boolean operator than a larger/smaller than symbol

**Comment by Limeboy0603** [comment]
https://github.com/Timwi/KtaneContent/pull/501
couldnt auto merge, have to make new pr

**Comment by Limeboy0603** [comment]
#504

**Comment by red031000** [comment]
no longer needed

**Pull request closed without merging**